core/test/qa_grc.cpp

```diff
@@ -9,20 +9,28 @@
 
 #include <boost/ut.hpp>
 
+#if defined(__clang__) && __clang_major__ >= 16
+// clang 16 does not like ut's default reporter_junit due to some issues with stream buffers and output redirection
+template<>
+auto boost::ut::cfg<boost::ut::override> = boost::ut::runner<boost::ut::reporter<>>{};
+#endif
+
```

Comment:
Is this still relevant since we use clang17?

Reply:
Yes, crashed on clang 17 for me